edge/values.yaml

```
@@ -47,52 +47,18 @@ edge:
       cpu: 100m
       memory: 128Mi
 
-    # There are a lot of options for ingress into a kubernetes cluster.  
-    # Change the values below if you want to use an ingress controller other than Voyager
-
-    # For instance, these values will work for nginx ingress
-
-    # apiVersion: extensions/v1beta1
-    # annotations:
-    #   nginx.ingress.kubernetes.io/ssl-passthrough: "true"
-    #   nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
-    #   nginx.ingress.kubernetes.io/backend-protocol: "https"
-
-    # rules:
-    #   - host: greymatter.development.deciphernow.com
-    #     http:
-    #       paths: 
-    #         - path: /
-    #           backend:
-    #             serviceName: edge
-    #             servicePort: 8080
-
   ingress:
-    apiVersion: voyager.appscode.com/v1beta1
+    # For ingress, you can specify which API verison  to use
+    apiVersion: extension/v1beta1
   
     # Specific annotations for your deployment
     # Choosing the ingress.appscode.com/type is important and depends on your environment type.
     #   - If using EKS, you need to configure the Type as LoadBalancder 
     #   - If using voyager, you need to configure the type as NodePort
     annotations: 
-      kubernetes.io/ingress.class: "voyager"
-      ingress.appscode.com/ssl-passthrough: "true"
-      ingress.appscode.com/type: NodePort
     
     # Provide the rule set you want your ingress controller to use
-    rules:
-      - tcp:
-          port: '80'
-          nodePort: "30001"
-          backend:
-            serviceName: edge
-            servicePort: 8080
-      - tcp:
-          port: '443'
-          nodePort: "30000"
-          backend:
-            serviceName: edge
-            servicePort: 8080
+    rules: 
```

Same here, except it needs []
```suggestion
    rules: []
```

---

Good catch!  thanks

---

I've pushed an update for this.